docs/getting-started/06-create-redis-service-instance.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Create a ServiceInstance for the Redis service
+type: Getting Started
+---
+
+To create a binding between the microservice and the Redis service, you must first create an [instance](https://svc-cat.io/docs/walkthrough/#step-4---creating-a-new-serviceinstance) of the previously provisioned service.
+
+## Steps
+
+Follows these steps:
+
+<div tabs name="steps" group="create-redis-instance">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Create a [ServiceInstance CR](https://svc-cat.io/docs/walkthrough/#step-4---creating-a-new-serviceinstance). You will provision the [Redis](https://redis.io/) service with its `micro` plan:
+
+   ```yaml
+   cat <<EOF | kubectl apply -f -
+   apiVersion: servicecatalog.k8s.io/v1beta1
+   kind: ServiceInstance
+   metadata:
+     name: redis-service
+     namespace: orders-service
+   spec:
+     serviceClassExternalName: redis
+     servicePlanExternalName: micro
+     parameters:
+       imagePullPolicy: Always
+   EOF
+   ```
+
+2. Check that the ServiceInstance CR was created. The last condition in the CR status should state `Ready True`:
+
+   ```bash
+   kubectl get serviceinstance redis-service -n orders-service -o=jsonpath="{range .status.conditions[*]}{.type}{'\t'}{.status}{'\n'}{end}"
+   ```
+
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+1. Go to **Service Management** > **Catalog** in the left navigation panel.
```

Comment:
```suggestion
1. In the left navigation panel, go to **Service Management** > **Catalog**.
```
See [this comment](https://github.com/kyma-project/kyma/pull/9296#discussion_r478392039).